plugins/mentions.md

```diff
@@ -93,6 +91,55 @@ tinymce.init({
 });
 ```
 
+### `mentions_item_type`
+
+This option allows you to specify what kind of user interface item to use when displaying the list of users. The `name` item will just display the user's name, while the `profile` item can also display an optional image and description. Depending on what this option is set to, additional properties may be required for the user object provided via [mentions_fetch](#mentions_fetch). Refer to the [User properties](#userproperties) table for more details.
+
+**Type:** `String`
+
+**Default Value:** `name`
+
+**Possible Values:** `name`, `profile`
+
+#### Example: Using `mentions_item_type`
+
+```js
+tinymce.init({
+  selector: 'textarea',
+  plugins: 'mentions',
+  mentions_item_type: 'profile'
+});
+```
+
+##### User properties
+
+What properties are required, optional and not available depends on the [mentions_item_type](#mentions_item_type) setting and the [mentions_select](#mentions_select) setting.
```

Comment:
```suggestion
Which properties are required, optional and not available depends on the [mentions_item_type](#mentions_item_type) setting and the [mentions_select](#mentions_select) setting.
```

Can we add a sentence in here just specifying what the table below is? In particular it's not fully clear whether `name` and `profile` refer to mentions item type or mentions select.

Reply:
Clarified this and described what refers to what.